submissions/xT0PERx/Popup-style/style.css

```diff
@@ -4,42 +4,126 @@
     flex-direction: row;
     justify-content: flex-end;
     font-size: 1em;
-    text-align: center;
     padding: 5px;
 }
 
+.container ol {
+    display: flex;
+    flex-flow: wrap row;
+}
+.container li {
+    display: inline-block;
+    width: 50px;
+    height: 50px;
+    margin: 5px;
+    text-align: center;
+}
+
 .container div {
     width: 80px;
     height: 80px;
     margin: 5px;
     cursor: pointer;
 }
 
+.main_tullbar {
+    position: relative;
+}
+
 img {
-    width: 40px;
-    height: 40px;
+    width: 32px;
+    height: 32px;
 }
 
-label img:hover {
-    border: 1px solid #1d92bd;
-    border-radius: 20%;
+.menu {
+    position: absolute;
+    background-image: url(./images/Menu.png);
+    width: 32px;
+    height: 32px;
+    cursor: pointer;
+    right: 4px;
+    top: 3px;
+}
+
+.check_input {
+    -webkit-appearance: none;
+}
+
+/*                                                      Normal */
+
+.mail p,
+.images p,
+.menu,
+.menu_more,       
+.home img,
+.avatar img {
+    text-decoration: none;
+    border: 1.5px solid transparent;
+}
+
+.menu {
+    border: 1.5px solid white
+}
+
+.mail,
+.images,
+.home,
+.avatar {
+    text-decoration: none;
+}
+.icons div {
```

Comment:
Try to avoid styling by HTML tag. It makes you, selector, more specific. Read more about specificity [Link#1](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), [Link#2](https://www.smashingmagazine.com/2007/07/css-specificity-things-you-should-know/)
Separate responsibilities: HTML - for markup, CSS classes - for styling, ID - for JS logic and linking inputs with labels.

Reply:
I understand what you are talking about. It turns out that every my html element to which I apply the css property must have its own class?
And another question, can I start to do this already with the next task? Or do I need to redo everything?

Reply:
Yes, each similar element must have a class. One example: [Link](https://9elements.com/bem-cheat-sheet/) Here some examples of naming element using BEM methodology
Leave or rewrite it's up to you.
But try to make an accessible popup(using keyboard).